vars/runBenchmarkTestScript.groovy

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/** Library to execute benchmark-test using opensearch-benchmark and opensearch-cluster-cdk
+ *
+ * @param Map args = [:] args A map of the following parameters
+ * @param args.bundleManifest <required> - OpenSearch bundle manifest url.
+ * @param args.insecure <required> - Force the security of the cluster to be disabled, default is false.
+ * @param args.workload <required> - Name of the workload that OpenSearch Benchmark should run, default is nyc_taxis.
+ * @param args.singleNode <required> - Create single node OS cluster, default is true.
+ * @param args.minDistribution <required> - Use min distribution of OpenSearch for cluster, default is false.
+ * @param args.useFiftyPercentHeap <required> - Use 50 percent of physical memory as heap, default is false.
+ * @param args.captureNodeStat <required> - Make opensearch-benchmark to capture node stats during run, default is false
+ * @param args.suffix <optional> - Suffix to be added to stack name for benchmark test.
+ * @param args.managerCount <optional> - Number of manager nodes in multi-node cluster, default is 3.
+ * @param args.dataCount <optional> - Number of data nodes in multi-node cluster, default is 2.
+ * @param args.clientCount <optional> - Number of client nodes in multi-node cluster, default is 0.
+ * @param args.ingestCount <optional> - Number of ingest nodes in multi-node cluster, default is 0.
+ * @param args.mlCount <optional> - Number of ml nodes in multi-node cluster, default is 0.
+ * @param args.workloadParams <optional> - Additional parameters for benchmark workload type.
```

Comment:
Maybe provide an example?

Reply:
Yeah but for users using this library in their jenkinsFile it can be useful.

Reply:
Added.